forgotten-sender/listing-02/main.go

```diff
@@ -0,0 +1,27 @@
+// Example program that performs serial work. It does not
+// launch any goroutines or suffer from leaks. Yet.
+package main
```

Comment:
This is good. We might want to report the leak like I did in the playground link I provided for the other example. Think about it.